xml/net_dns.xml

```diff
@@ -776,7 +776,7 @@
    <command>dig</command> programs, which should return
    <systemitem class="domainname">localhost</systemitem> as the default server
    with the address <systemitem class="ipaddress">127.0.0.1</systemitem>. If
-   this is not the case, <filename>/etc/resolv.conf</filename> probably
+   this is not the case, <filename>/var/run/netconfig/resolv.conf</filename> probably
```

Comment:
`dig` and `host` are using /etc/resolv.conf.

Reply:
Thanks, done.